en/mozorg/about/history.ftl

```diff
@@ -0,0 +1,101 @@
+
+# Variables:
+#   $stevecase (url) link to https://www-archive.mozilla.org/stevecase.html
+history-by-creating-an-open-community = By creating an open community, the { -brand-name-mozilla } project had become <a href="{ $stevecase }">larger than any one company</a>.
+
+# Variables:
+#   $mission (url) link to https://www-archive.mozilla.org/mission.html
+#   $browsers (url) link to http://www.oreillynet.com/pub/a/mozilla/2002/09/12/mozilla_browsers.html
+#   $bugzilla (url) link to https://www.bugzilla.org
+#   $projects (url) link to https://www.mozilla.org/firefox/
+history-community-members-got-involved = Community members got involved and expanded the scope of the project’s <a href="{ $mission }">original mission</a> — instead of just working on { -brand-name-netscape }’s next browser, people started creating <a href="{ $browsers }">a variety of browsers</a>, <a href="{ $bugzilla }">development tools</a> and a range of other <a href="{ $projects }">projects</a>.
+
+history-people-contributed-to-mozilla = People contributed to { -brand-name-mozilla } in different ways, but everyone was passionate about creating free software that would enable people to have a choice in how they experienced the internet.
+
+# Variables:
+#   $mozilla1 (url) link to http://www.mozillazine.org/articles/article2278.html
+history-after-several-years-of-development = After several years of development, <a href="{ $mozilla1 }">{ -brand-name-mozilla } 1.0</a>, the first major version, was released in 2002. This version featured many improvements to the browser, email client and other applications included in the suite, but not many people were using it.
+
+# Variables:
+#   $over90 (url) link to http://www.onestat.com/html/aboutus_pressbox4.html
+history-by-2002-well-over-90 = By 2002, <a href="{ $over90 }">well over 90% of internet users</a> were browsing with { -brand-name-ie }.
+
+# Variables:
+#   $charter (url) link to https://www-archive.mozilla.org/projects/firefox/charter.html
+history-not-many-people-noticed = Not many people noticed at the time, but the first version of Phoenix (later renamed to { -brand-name-firefox }) was also released by { -brand-name-mozilla } community members that year with the goal of providing the <a href="{ $charter }">best possible browsing experience</a> to the widest possible set of people.
+
+# Variables:
+#   $foundation (url) link to https://blog.mozilla.org/press/2003/07/mozilla-org-announces-launch-of-the-mozilla-foundation-to-lead-open-source-browser-efforts/
+history-in-2003-the-mozilla-project = In 2003, the { -brand-name-mozilla } project created the { -brand-name-mozilla } Foundation, an <a href="{ $foundation }">independent non-profit organization</a> supported by individual donors and a variety of companies.
```

Comment:
`the { -brand-name-mozilla } Foundation` -> `the { -brand-name-mozilla-foundation }`